src/app/layout.tsx

```diff
@@ -1,22 +1,25 @@
-import './globals.css'
-import type { Metadata } from 'next'
-import { Inter } from 'next/font/google'
+import type { Metadata } from "next";
+import { Inter } from "next/font/google";
+import SessionAuthProvider from "../../context/SessionAuthProvider";
+import "./globals.css";
 
-const inter = Inter({ subsets: ['latin'] })
+const inter = Inter({ subsets: ["latin"] });
 
 export const metadata: Metadata = {
-  title: 'Create Next App',
-  description: 'Generated by create next app',
-}
+  title: "Regresemos a nuestra casa",
+  description: "Generated by create next app",
```

Comment:
Could you please edit this?

```suggestion
  description: "Created by Generación desafiante",
```

Reply:
fixed